tvtk/pyface/ui/qt4/scene.py

```diff
@@ -338,7 +339,7 @@ def __init__(self, parent=None, **traits):
         super(Scene, self).__init__(parent, **traits)
 
         # Setup the default picker.
-        self.picker = picker.Picker(self)
+        self.picker = picker.Picker(weakref.proxy(self))
```

Comment:
I have added an issue https://github.com/enthought/mayavi/issues/214 for this as I do not know why `renwin`, and other `pickers (point picker, cell picker etc)` are not traits. @prabhuramachandran ?